hack/make-rules/test-cmd-util.sh

```diff
@@ -1813,7 +1813,9 @@ run_secrets_test() {
 
   ### Create a docker-registry secret in a specific namespace
   # Pre-condition: no SECRET exists
-  kube::test::get_object_assert 'secrets --namespace=test-secrets' "{{range.items}}{{$id_field}}:{{end}}" ''
```

Comment:
this looks like it is removing the precondition checks entirely for non-federation runs